src/lib/operators/get_table.cpp

```diff
@@ -180,6 +189,9 @@ std::shared_ptr<const Table> GetTable::_on_execute() {
 
       *output_chunks_iter = std::make_shared<Chunk>(std::move(output_segments), stored_chunk->mvcc_data(),
                                                     stored_chunk->get_allocator(), std::move(output_indexes));
+      if (chunk_ordered_by) {
+        (*output_chunks_iter)->set_ordered_by(*chunk_ordered_by);
```

Comment:
What if the column was pruned and the then clause in line 177 was never executed? Unless I am missing something, this is also missing a test.

Reply:
Good catch. Should be solved. Test added which failed without the change.